sdk/vaa/structs.go

```diff
@@ -245,6 +245,8 @@ func (c ChainID) String() string {
 		return "holesky"
 	case ChainIDPolygonSepolia:
 		return "polygon_sepolia"
+	case ChainIDMonadDevnet:
+		return "monad_devnet"
```

Comment:
Why wouldn't this just be "monad"?

Reply:
This is to avoid a conflict with any future potential testnet run in parallel.